samples/bluetooth/fast_pair/locator_tag/Kconfig.sysbuild

```diff
@@ -10,6 +10,11 @@ config BT_FAST_PAIR
 config APP_DFU
 	bool "Enable MCUmgr OTA DFU over Bluetooth"
 	default y
-	depends on BOOTLOADER_MCUBOOT
+	depends on BOOTLOADER_MCUBOOT || SOC_SERIES_NRF54HX
+	help
+	  This option enables the DFU functionality in this application.
+	  The DFU solution is supported for the MCUboot-based targets
+	  (BOOTLOADER_MCUBOOT Kconfig) and for the SUIT-based targets
+	  (SOC_SERIES_NRF54HX Kconfig).
```

Comment:
I prefer to have this option enabled by default as most configurations should have the DFU functionality enabled (you should have good reasons to disable this option, e.g. not enough memory)

Reply:
Enabling it by default is reasonable if we know that we support bootloader (in our case MCUboot).
nRF54H is a "special" use-case for now. Information if SUIT is enabled is not available at sysbuild level. We provide configuration required for the DFU support only for the nRF54H20 DK (other nRF54H-based boards would not have SUIT configured out-of-the-box for locator_tag sample). Because of that I would suggest to avoid enabling this option by default for nRF54H-based boards apart from the DK. I think that disabled DFU is better than failures during build or runtime.

Reply:
Of course other SoC series could still enable it by default

Reply:
Discussed offline. We will rely soon on sysbuild Kconfig option for SUIT, for now let's leave as is.